tests/optimizers/test_objective_func_with_kwargs.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Import modules
+import pytest
+import numpy as np
+
+# Import from package
+from pyswarms.single import GlobalBestPSO, LocalBestPSO
+from pyswarms.utils.functions.single_obj import rosenbrock_func
+
+
```

Comment:
Thanks for the awesome tests! Just a few more tests to write:

1. Can you please test if an error is raised when `*args`/`**kwargs` is passed but it's not defined in the objective function itself? (e.g. I passed `c` and `d` but rosenbrock accepts `a` and `b`).
2. Can you also test if error is raised when not all `*args`/`**kwargs` are supplied? (e.g. I only passed `a` but rosenbrock accepts both `a` and `b`)
3. Test what happens if I passed more than one `*args`/`**kwargs` than necessary.